### PR TITLE
18014 - Update payment instructions

### PR DESCRIPTION
**Pull request by rodrigo-barraza**
*Issue #:18014 *
https://github.com/bcgov/entity/issues/18014 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: report-api/report-templates/eft_instructions.html [hunk 1]
```diff
@@ -187,47 +187,45 @@ <h3 class="font-bold">EFT Banking Details</h2>
       <td>Canadian Imperial Bank of Commerce</td>
```
Comment (ochiu) [inline report-api/report-templates/eft_instructions.html:187]: Should be Royal Bank of Canada?

File: report-api/report-templates/eft_instructions.html [hunk 2]
```diff
@@ -187,47 +187,45 @@ <h3 class="font-bold">EFT Banking Details</h2>
...
+    <p><span class="font-bold">Include</span>:</p>
+    <ul>
+      <li>Date payment was made</li>
+      <li>Amount of payment</li>
+      <li>BC Registries Online and Services Account Name (and branch name if you have one)</li>
+      <li>BC Registries Online and Services Account Number: <span class="font-bold">12345</span></li>
+    </ul>
+    <p>Our staff will use the provided information to establish a linkage between your bank and your BC Registries Online and Services account. This means you won't need to email us every time for subsequent payments.</p>
+    <p>If you choose to use a different bank account for subsequent payments, kindly resend the details via email to ensure accurate processing by our system.</p>
+    <p>If you have any questions or require further assistance, you can reach us at 1-8777-526-1526</p>
```
Comment (ochiu) [inline report-api/report-templates/eft_instructions.html:228]: 1-8777-526-1526

too many 7s

File: report-api/report-templates/eft_instructions.html [hunk 3]
```diff
@@ -187,47 +187,45 @@ <h3 class="font-bold">EFT Banking Details</h2>
       <td>Canadian Imperial Bank of Commerce</td>
     </tr>
     <tr>
-      <td>Bank Address:</td>
+      <td>Branch Address:</td>
       <td>1025 West Georgia St., Vancouver BC V6E 3N9</td>
```
Comment (ochiu) [inline report-api/report-templates/eft_instructions.html:191]: UX as Street vs St.

**Comment by seeker25** [comment]
Reopen, when we have the data for this

**Pull request closed without merging**